armi/reactor/converters/uniformMesh.py

```diff
@@ -390,12 +390,13 @@ def adjointFluxGetter(block, _paramNames):
             _setStateFromOverlaps(
                 aSource, aDest, paramSetter, paramGetter, self.blockParamNames
             )
-            # Now recalculate derived params with the mapped flux to minimize
-            # potential numerical diffusion (e.g. control rod tip into large coolant)
-            for b in aDest:
-                globalFluxInterface.calcReactionRates(
-                    b, destReactor.core.p.keff, destReactor.core.lib
-                )
+            if destReactor.core.lib is not None:
```

Comment:
needs explanation. It's unclear from reading the code here why this would happen and also why it's ok to just skip it if the library is none.

Reply:
That a lib might not exist should be made clear from the Core class itself.